routes/sessionRoutes.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");
+const passport = require("passport");
+const router = express.Router();
+
+const {
+  logonShow,
+  registerShow,
+  registerDo,
+  logoff,
+} = require("../controllers/sessionController");
+
+router.route("/register").get(registerShow).post(registerDo);
+router
```

Comment:
Take a look to see what issue this could be causing by trying to run other routes.